packages/builder/src/pages/builder/app/[application]/settings/oauth2/MoreMenuRenderer.svelte

```diff
@@ -45,5 +45,5 @@
 </ActionMenu>
 
 <Modal bind:this={modal}>
-  <OAuth2ConfigModalContent config={row} />
+  <OAuth2ConfigModalContent config={{ ...row }} />
```

Comment:
This will make a copy of the row, otherwise the in-memory one will be modified before being saved